src/tokens.js

```diff
@@ -259,6 +286,30 @@ const tokens = {
     return null;
   },
 
+  /**
+   * Validates if already has a token with same name or symbol added in the wallet
+   *
+   * @param {string} name Token name to search
+   * @param {string} symbol Token symbol to search
+   *
+   * @return {Object|null} Token if name or symbol already exists, else null
+   *
+   * @memberof Tokens
+   * @inner
+   */
+  tokenInfoExists(name, symbol) {
+    const tokens = this.getTokens();
+    for (const token of tokens) {
+      if (token.name === name) {
+        return {token, key: 'name'};
+      }
+      if (token.symbol === symbol) {
```

Comment:
The same here. A simple comparison seems too simple and trivial to be bypassed.

Reply:
I've done the strip cleanup, do you think we should do more? Maybe begin with this only?

Reply:
Maybe we should create a cleanup method. I would be concerned about:
1) Uppercase vs Lowercase
2) Spaces at the beginning or at the end
3) Multiples spaces in the middle (in face, should we allow any space at all in the symbol?)
4) Special characters with unexpected results.

Reply:
Same as conversation https://github.com/HathorNetwork/hathor-wallet-lib/pull/26#discussion_r330072548